buildSrc/src/main/java/Dependencies.kt

```diff
@@ -6,7 +6,8 @@ object Versions {
   val KotlinCompiler = System.getProperty("square.kotlinVersion") ?: "1.9.10"
 
   const val AndroidXTest = "1.5.0"
-  const val Compose = "1.5.3"
+  const val Compose = "1.7.0-beta05"
+  const val ComposeCompiler = "1.5.3"
```

Comment:
I didn't want to attempt to update kotlin at the same time, so I left the compose compiler version